### DISCUSSION TITLE
GPG Key for falcon-sensor RPM package files?

### DISCUSSION THREAD

**Discussion by vcleniukBHS** [CrowdStrike/community#68] [Q&A]
Hi,

We're incorporating CrowdStrike falcon-sensor packages in our corporate RHEL Satellite servers.  Is there a way to pull the GPG keys for each falcon-sensor RPM package from the CrowdStrike cloud?  We just need to plug the GPG keys into RHEL Satellite.

VC

**Comment by redhatrises** [comment]
Hello,

Unfortunately, there is no way to do this at the moment right now. It is definitely sorely needed. I can only suggest to work with CS support to request this feature.

**Comment by carlosmmatos** [comment]
At the moment, you can just download the GPG key from the UI and upload it that way. We do something similar on our Ansible collection since the key doesn't change often.

[falcon-sensor.gpg](https://github.com/CrowdStrike/ansible_collection_falcon/blob/main/roles/falcon_install/files/falcon-sensor.gpg)

Also, @redhatrises is right, you should really bug CS support to get this added as a feature request ;)

**Comment by hakong** [comment]
Hi, same issue here. Would like to synchronize a yum repo to our Satellite server.

Can we get a yum repository with signed packages and a gpg key?

**Comment by fin-major** [comment]
I was able to get the GPG Key from the install script:
https://raw.githubusercontent.com/crowdstrike/falcon-scripts/main/bash/install/falcon-linux-install.sh
